### PR TITLE
nixos/btrbk: allow instances without timers and simplify

### PR DESCRIPTION
**Pull request by oxalica**
###### Description of changes

This allows btrbk instances with without a triggering timer by setting `onCalendar` to `null`. This is useful for manual-only btrbk backup settings.

Also my GPG key are rotated. Commits in this PR are signed by the new key. Currently both the old and the new keys are valid on GitHub (https://github.com/oxalica.gpg)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: nixos/modules/services/backup/btrbk.nix [hunk 1]
```diff
@@ -1,18 +1,21 @@
 { config, pkgs, lib, ... }:
+
+with lib;
```
Comment (ncfavier) [inline nixos/modules/services/backup/btrbk.nix:3]: There are arguments against this in https://github.com/NixOS/nixpkgs/pull/101139

Comment (oxalica) [reply nixos/modules/services/backup/btrbk.nix]: Fixed.

**Comment by ncfavier** [comment]
@ofborg test btrbk-no-timer

**Review by ncfavier — APPROVED** [review]

**Pull request merged**